lib/cdo/poste.rb

```diff
@@ -273,7 +273,7 @@ class DeliveryMethod
       hadi_partovi@code.org
       survey@code.org
       facilitators@code.org
-      tanya_parker@code.org
+      volunteers@code.org
```

Comment:
I wonder if this should be `volunteer@code.org`?  Sending to/from the plural might feel as though it's to/from all the volunteers that we have?  I guess we have precedent with `facilitators` though...